client/my-sites/checkout/composite-checkout/components/item-variation-picker/index.tsx

```diff
@@ -1,9 +1,19 @@
 import { FunctionComponent } from 'react';
+import { ItemVariationDropDown } from './item-variation-dropdown';
 import { ItemVariationRadioButtons } from './item-variation-radio-buttons';
+import { ItemVariationPickerPlaceholder } from './placeholder';
 import type { ItemVariationPickerProps } from './types';
 
-export const ItemVariationPicker: FunctionComponent< ItemVariationPickerProps > = ( props ) => (
-	<ItemVariationRadioButtons { ...props } />
-);
+export const ItemVariationPicker: FunctionComponent< ItemVariationPickerProps > = ( props ) => {
+	if ( props.isLoading ) {
```

Comment:
this is a temporary construct, once the A/B Test is complete it can be safely removed.